app/services/action_queue.rb

```diff
@@ -172,14 +165,6 @@ def fake_card_info
     def get_payment_account
       "GoCardless #{data[:currency]}"
     end
-
-    def action_fields
-      {
-        recurring_id:      @action.member_id,
```

Comment:
Tuuli unless I'm mistaken this is something that should have been removed when Omar refactored the recurring donations integration to make use of the fact AK's donation push API can now handle recurring donations, given passing recurring_id etc. as an action field was our hacky workaround before that refactor.